js/collections/Followers.js

```diff
@@ -7,18 +7,20 @@ module.exports = Collection.extend({
   /* we have to use the older style for this collection, the ES6 style creates a bug where models
```

Comment:
None of these changes to followers should have been in the PR, that was an accidental inclusion of a half-done refactor. I'll take it out, it should be done as a separate PR.